HousingRepairsOnlineApi/Properties/launchSettings.json

```diff
@@ -51,7 +51,8 @@
         "SOR_CONFIGURATION_LEASEHOLD": "[{\"value\":\"livingAreas\",\"display\":\"Living areas\",\"options\":[{\"value\":\"window\",\"display\":\"Window\",\"options\":[{\"value\":\"Emergency\",\"display\":\"Smashed window(s)\"},{\"value\":\"Emergency\",\"display\":\"Window stuck open\"},{\"value\":\"ContactUs\",\"display\":\"Window stuck shut\"},{\"value\":\"ContactUs\",\"display\":\"Condensation\"}]}]},{\"value\":\"outside\",\"display\":\"Outside\",\"options\":[{\"value\":\"roofIncludingInsulationAndShedRoof\",\"display\":\"Roof, including insulation and shed roof\",\"options\":[{\"value\":\"ContactUs\",\"display\":\"Leak\"},{\"value\":\"tilesHaveComeOff\",\"display\":\"Tiles have come off\",\"sorCode\":\"<enter sor code>\",\"priority\":\"2\"}]},{\"value\":\"chimney\",\"display\":\"Chimney \",\"options\":[{\"value\":\"ContactUs\",\"display\":\"Needs repointing or repairing\"}]},{\"value\":\"gatesAndPathways\",\"display\":\"Gates and pathways\",\"options\":[{\"value\":\"concretePathAroundTheProperty\",\"display\":\"Concrete path around the property\",\"sorCode\":\"<enter sor code>\",\"priority\":\"2\"}]},{\"value\":\"propertyWalls\",\"display\":\"Property walls\",\"options\":[{\"value\":\"missingOrLooseBrickwork\",\"display\":\"Missing or loose brickwork\",\"sorCode\":\"<enter sor code>\",\"priority\":\"2\"}]},{\"value\":\"ContactUs\",\"display\":\"Door, including shed and outhouse\"}]}]",
-        "ALLOWED_APPOINTMENT_SLOTS": "[ { \"startTime\": \"08:00:00\", \"endTime\": \"12:00:00\" }, { \"startTime\": \"12:00:00\", \"endTime\": \"18:00:00\" } ]"
+        "ALLOWED_APPOINTMENT_SLOTS": "[ { \"startTime\": \"08:00:00\", \"endTime\": \"12:00:00\" }, { \"startTime\": \"12:00:00\", \"endTime\": \"18:00:00\" } ]",
+        "REPAIR_DAYS_MAPPING": "[ { \"Priority\": \"2\", \"NumberOfDays\": \"30\" }, { \"Priority\": \"3\", \"NumberOfDays\": \"130\" } ]"
```

Comment:
I don't think that the name `REPAIR_DAYS_MAPPING` clearly reflects what the value represents, particularly as it doesn't include `PRIORITY` in the name, i.e. in could be taken to mean a mapping between repair types and the days on which they can be scheduled.
I think a more representative name would be `REPAIR_PRIORITY_TO_DAYS`.

With this view, an equivalent renaming should be made throughout the codebase, i.e. `RepairDayWindowHelper`